mp/src/game/client/momentum/ui/HUD/hud_stickybombs.cpp

```diff
@@ -67,19 +120,66 @@ void CHudStickybombs::OnThink()
         return;
 
     const auto pStickylauncher = static_cast<CMomentumStickybombLauncher *>(pStickyPtr);
-
     const auto iStickybombs = pStickylauncher->GetStickybombCount();
-    if (mom_hud_sj_stickycount_autohide.GetBool())
+
+    for (int i = 0; i < MOM_WEAPON_STICKYBOMB_COUNT; i++)
     {
-        m_pStickyLabel->SetVisible(iStickybombs > 0);
+        if (i < iStickybombs)
+        {
+            if (!m_Stickyboxes[i]->HasAnimationStarted())
+            {
+                m_Stickyboxes[i]->StartAnimation(m_PreArmColor, m_ArmedColor);
+            }
+
+            const auto pSticky = pStickylauncher->GetStickyByCount(i);
+            bool bCanExplode = !pSticky || pSticky->CanExplode();
+
+            m_Stickyboxes[i]->SetEnabled(bCanExplode);
+            if (!bCanExplode)
+                m_Stickyboxes[i]->SetBgColor(m_DisabledColor);
+        }
+        else if (m_Stickyboxes[i]->HasAnimationStarted())
+        {
+            if (i > 0 && i >= iStickybombs)
+            {
+                // moving into left-most box, so swap positions
+                V_swap(m_Stickyboxes[i], m_Stickyboxes[0]);
+                InvalidateLayout();
+            }
+
+            m_Stickyboxes[i]->StopAnimation();
+            m_Stickyboxes[i]->SetBgColor(m_NoStickyColor);
+        }
     }
-    else
+}
+
+void CHudStickybombs::ApplySchemeSettings(IScheme *pScheme)
+{
+    BaseClass::ApplySchemeSettings(pScheme);
+    SetBgColor(m_BgColor);
+
+    for (int i = 0; i < MOM_WEAPON_STICKYBOMB_COUNT; i++)
     {
-        m_pStickyLabel->SetVisible(true);
+        m_Stickyboxes[i]->SetBgColor(m_NoStickyColor);
+        m_Stickyboxes[i]->SetBorder(pScheme->GetBorder("ButtonDepressedBorder"));
```

Comment:
You can alias a border to another one in the scheme. I would recommend doing something like:

```suggestion
        m_Stickyboxes[i]->SetBorder(pScheme->GetBorder("StickyboxBorder"));
```

And then in `ClientScheme.res`:

```
. . .

Borders
{
    . . .

    StickyboxBorder ButtonDepressedBorder

    . . .
}
```

Reply:
I tried this but one thing I noticed is that `ClientScheme.res` doesn't seem to be able to reference values in `SourceScheme.res` like this. I have to place this in either `SourceScheme.res` or in the "BaseSettings" section of `ClientScheme.res`.

Also noticed this when I was trying to change the speedometer increase/decrease/normal colors to "MomentumBlue"/"MomentumRed"/etc.

Lastly, doing it this way makes the borders a different color for some reason:
![image](https://user-images.githubusercontent.com/9014762/93661570-9cc69400-fa0d-11ea-88c7-d31b8828ec44.png)
s o r s e

Reply:
Nevermind I figured this out. The border shown above is the default one when `pScheme->GetBorder` doesn't find it.

I put it in the border section and had it alias `FrameBorder` instead of `ButtonDepressedBorder`. I'm guessing because `ButtonDepressedBorder` is an alias of `FrameBorder` it just doesn't work? Weird.